postreise/plot/plot_bar_generation_stack.py

```diff
@@ -98,13 +98,17 @@ def plot_bar_generation_stack(
     mi = ModelImmutables(s_list[0].info["grid_model"])
     type2color = mi.plants["type2color"]
     type2label = mi.plants["type2label"]
-    type2hatchcolor = mi.plants["type2hatchcolor"]
-    if t2c:
-        type2color.update(t2c)
-    if t2l:
-        type2label.update(t2l)
-    if t2hc:
-        type2hatchcolor.update(t2hc)
```

Comment:
I guess we still want to keep these updates for user specified color/label, right?

Reply:
Yes. Not sure why I removed these lines.

Reply:
Fixed